utils/config.env

```diff
@@ -41,10 +41,10 @@ KAFKA_BRANCH=2.4
 #   REPOSITORY=<account>.dkr.ecr.us-west-2.amazonaws.com/confluentinc   # confluentinc repository at the specified ECR registry, images must be pulled separately
 #####################################################
 
-CP_VERSION_MAJOR=5.5
-CP_VERSION_FULL=5.5.0
-GIT_BRANCH=master
-JAR_VERSION=5.5.0-SNAPSHOT
+CP_VERSION_MAJOR="$CONFLUENT_MAJOR.$CONFLUENT_MINOR"
+CP_VERSION_FULL="$CONFLUENT_MAJOR.$CONFLUENT_MINOR.$CONFLUENT_PATH"
```

Comment:
Is this supposed to be `CONFLUENT_PATCH`?

Can you print these all out to verify they have meaningful values (ie no other spelling issues)?

Reply:
I ran demo tests so i'm not sure how this got by unless I inserted a typo prior to committing somehow

Reply:
printed out the calculated values and added a comment below